src/Microsoft.Crank.Controller/ScriptConsole.cs

```diff
@@ -13,27 +13,47 @@ public class ScriptConsole
 
         public void Log(params object[] args)
         {
-            Console.WriteLine(String.Join(" ", args.Select(x => x.ToString())));
+            if (args == null || args.Length == 0)
+            {
+                return;
+            }
+
+            Console.WriteLine(String.Join(" ", args.Select(x => x?.ToString())));
         }
 
         public void Info(params object[] args)
         {
+            if (args == null || args.Length == 0)
+            {
+                return;
+            }
+
             Console.ForegroundColor = ConsoleColor.Green;
-            Console.WriteLine(String.Join(" ", args.Select(x => x.ToString())));
+            Console.WriteLine(String.Join(" ", args.Select(x => x?.ToString())));
```

Comment:
Can't you just use `string.Join(string, params object[])` directly? It handles `null`.